src/components/projecteditor/index.js

```diff
@@ -36,6 +37,15 @@ const mapDispatchToProps = (dispatch) => {
         closeTransactionsHistoryPanel() {
             dispatch(sidePanelsActions.closeTransactionsHistoryPanel())
         },
+        toggleFileSystemPanel() {
+            dispatch(sidePanelsActions.toggleFileSystemPanel())
+        },
+        openFileSystemPanel() {
```

Comment:
openFileSystemPanel and closeFileSystemPanel functions and actions does not seems to be used, so please remove them.